projects/VitalRecord/VitalRecord_constructors.cs

```diff
@@ -70,10 +70,7 @@ public VitalRecord(string record, bool permissive = false)
                         FhirJsonParser parser = new FhirJsonParser(parserSettings);
                         Bundle = parser.Parse<Bundle>(record);
                     }
-
-                    // Validate the partial dates.
-                    ValidatePartialDates(Bundle);
```

Comment:
For now, can this just be commented out saying we are reevaluating the partial date handling and need to remove this check to get the natality parsing working?